syncany-lib/src/main/java/org/syncany/operations/daemon/ControlServer.java

```diff
@@ -59,6 +59,15 @@ public ControlServer() {
 		this.controlFileTailer = new Tailer(controlFile, this, 1000, true);
 		this.eventBus = LocalEventBus.getInstance();		
 	}
+	
+	/**
+	 * Constructor required for unit testing, as you can inject mocks in this way.
+	 */
+	public ControlServer(File ctrlFile, Tailer ctrTailer, LocalEventBus eventBus) {
```

Comment:
You should probably mark this as `@Deprecated` to indicate that this constructor should not be used. In most IDEs this will be more obvious than just a Javadoc comment.